juice-containers.yml

```diff
@@ -327,3 +346,254 @@ jobs:
+    - task: gen-container
+      image: container
+      config:
+        platform: linux
+        inputs:
+        - name: ci
+        outputs:
+        - name: recombined
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            cp ci/container/ubuntu/Dockerfile recombined/Dockerfile
+            cp ci/container/ubuntu/pkg.sh.default recombined/pkg.sh
+            cp ci/container/escalate.sh recombined/escalate.sh
+
+    - put: container-ubuntu-default
+      params:
+        build: recombined
+        tag_as_latest: true
+
+  - name: build-ubuntu-native
+    serial_groups: [constructors]
+    build_logs_to_retain: 2
+    plan:
+    - aggregate:
+      - get: ci
+        resource: git-continuous-integration
+        trigger: true
+      - get: container
+    - task: gen-container
+      image: container
+      config:
+        platform: linux
+        inputs:
+        - name: ci
+        outputs:
+        - name: recombined
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            cp ci/container/ubuntu/Dockerfile recombined/Dockerfile
+            cp ci/container/ubuntu/pkg.sh.native recombined/pkg.sh
+            cp ci/container/escalate.sh recombined/escalate.sh
+
+    - put: container-ubuntu-native
+      params:
+        build: recombined
+        tag_as_latest: true
+
+  - name: build-ubuntu-cuda
+    serial_groups: [constructors]
+    build_logs_to_retain: 2
+    plan:
+    - aggregate:
+      - get: ci
+        resource: git-continuous-integration
+        trigger: true
+      - get: container
+    - task: gen-container
+      image: container
+      config:
+        platform: linux
+        inputs:
+        - name: ci
+        outputs:
+        - name: recombined
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            cp ci/container/ubuntu/Dockerfile recombined/Dockerfile
+            cp ci/container/ubuntu/pkg.sh.cuda recombined/pkg.sh
+            cp ci/container/escalate.sh recombined/escalate.sh
+
+    - put: container-ubuntu-cuda
+      params:
+        build: recombined
+        tag_as_latest: true
+
+  - name: build-ubuntu-opencl
+    serial_groups: [constructors]
+    build_logs_to_retain: 2
+    plan:
+    - aggregate:
+      - get: ci
+        resource: git-continuous-integration
+        trigger: true
+      - get: container
+    - task: gen-container
+      image: container
+      config:
+        platform: linux
+        inputs:
+        - name: ci
+        outputs:
+        - name: recombined
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            cp ci/container/ubuntu/Dockerfile recombined/Dockerfile
+            cp ci/container/ubuntu/pkg.sh.opencl recombined/pkg.sh
+            cp ci/container/escalate.sh recombined/escalate.sh
+
+    - put: container-ubuntu-opencl
+      params:
+        build: recombined
+        tag_as_latest: true
+
```

Comment:
are you sure all of these were missing in the previous one? see the comment above, autogeneration should take care of this.